composer.json

```diff
@@ -148,7 +148,8 @@
         "phpdocumentor/reflection-docblock": "<3.0|>=3.2.0,<3.2.2",
         "phpdocumentor/type-resolver": "<0.3.0|1.3.*",
         "ocramius/proxy-manager": "<2.1",
-        "phpunit/phpunit": "<5.4.3"
+        "phpunit/phpunit": "<5.4.3",
+        "psr/container": "1.1.0"
```

Comment:
I don't think we need this. Composer will do its job at selecting the latest version.

Reply:
All right.